packages/builder/builder-doc/docs/en/plugins/plugin-swc.mdx

```diff
@@ -52,13 +54,30 @@ Now you can use SWC transformation and minification in your project.
 
 ```ts
 type PluginConfig = {
+  typescript?: boolean;
+  legacyDecorators?: boolean;
   presetReact?: ReactConfig;
   presetEnv?: EnvConfig;
   jsMinify?: boolean | JsMinifyOptions;
   extensions?: Extensions;
 };
 ```
 
+### typescript
+
+- Type: `boolean`
+- Default: `true`
+
+Whether to enable TypeScript compilation.
```

Comment:
I think in almost cases we will enable TypeScript compilation. In which scenario will the user set `typescript` to `false`?

Reply:
For now, TC39 decorator is only valid if TypeScript parsing is disabled, and disable TypeScript parsing will improve a little bit performance

Reply:
Deleted this 2 options, I'll figure out if I can manually invoke decorator transform API